### ISSUE TITLE
formatting

### ISSUE THREAD

**Issue by winthrop-polk** [t3hnar/CmdSupport#34]
Hi,

Just wanted to let you know I've been using this one: https://plugins.jetbrains.com/plugin/265-batch-scripts-support

But something weird happened today and now every time I try to input a bash comment with "::" the plugin goes crazy and stops me from typing. I tried a lot of different things (post: https://intellij-support.jetbrains.com/hc/en-us/community/posts/360004274179-IntelliJ-Stopped-Editor-Working)

Long story short I switch to this CMD Support instead and the problem is fixed... however the format highlighting was much much better in the old version; I just wanted to let you guys know, maybe you can implement some of that old functionality.

**Comment by t3hnar** [comment]
hello @winthrop-polk, unfortunately I don't have time to support this project anymore :(

**Comment by patricknelson** [comment]
Hi @t3hnar, that's understandable. I happened upon it myself because PhpStorm recommended it since it found a "compatible plugin" for `.bat` files. Can you remove it from the Jetbrains store or otherwise flag it as incompatible so that it's no longer recommended in modern versions of the IDE?
